torchmetrics/text/bert.py

```diff
@@ -130,7 +132,7 @@ class BERTScore(Metric):
 
     def __init__(
         self,
-        model_name_or_path: Optional[str] = None,
+        model_name_or_path: str,
```

Comment:
```suggestion
        model_name_or_path: Optional[str] = None,
```

After the fix at #760, `model_name_or_path` argument should remain optional